kernel_gateway/services/notebooks/handlers.py

```diff
@@ -20,19 +20,15 @@ class NotebookAPIHandler(TokenAuthorizationMixin, CORSMixin, tornado.web.Request
     execute_result = None
     stream_messages = []
     error_message = None
-    _assignment_statements = {'r': "REQUEST <- '{}'",
-        None : "REQUEST = '{}'"}
 
     def initialize(self, sources, kernel_pool, kernel_name):
         self.kernel_pool = kernel_pool
         self.sources = sources
         self.kernel_name = kernel_name
 
-    def _request_assignment_for_lang(self, kernel_name, expression):
-        try:
-            statement = self._assignment_statements[kernel_name]
-        except KeyError:
-            statement = self._assignment_statements[None]
+    def _format_request(self, expression):
+        expression = expression.replace('\"', '\\"')
```

Comment:
A comment in here about why we're doing this escaping will probably help future contributors.